base/applications/msconfig_new/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ list(APPEND CPP_SOURCE
 
 add_rc_deps(msconfig.rc ${CMAKE_CURRENT_SOURCE_DIR}/res/msconfig.ico)
 add_executable(msconfig_new ${C_SOURCE} ${CPP_SOURCE} msconfig.rc)
-add_pch(msconfig_new precomp.h CPP_SOURCE)
+add_pch(msconfig_new precomp.h CPP_SOURCE NO_UNITY_BUILD)
```

Comment:
lgtm,
I'm assuming you considered the `NO_UNITY_BUILD` ones on a per-case base so I won't comment on every single one.

Reply:
Basically these are just skipped for now until further examination. We may still be able to get some of them to join the.. union ;)